src/main/resources/sam.conf

```diff
@@ -243,3 +243,52 @@ janitor {
   trackResourceTopicId = ${?JANITOR_TRACK_RESOURCE_TOPIC_ID}
 }
 
+terra.support.emails = [
+  // dynamically configuring lists is hard, so 10 slots are provided for support emails
+  // add more if more than 10 are ever needed
+  ${?TERRA_SUPPORT_EMAIL_0}
+  ${?TERRA_SUPPORT_EMAIL_1}
+  ${?TERRA_SUPPORT_EMAIL_2}
+  ${?TERRA_SUPPORT_EMAIL_3}
+  ${?TERRA_SUPPORT_EMAIL_4}
+  ${?TERRA_SUPPORT_EMAIL_5}
+  ${?TERRA_SUPPORT_EMAIL_6}
+  ${?TERRA_SUPPORT_EMAIL_7}
+  ${?TERRA_SUPPORT_EMAIL_8}
+  ${?TERRA_SUPPORT_EMAIL_9}
+]
+
+resourceAccessPolicies {
+  resource_type_admin {
```

Comment:
What's an example for doing this for services (Leo, Rawls)? Wondering if our existing roles will work, or if it would be better to create some new ones for service-to-service communication..

Reply:
my intent was to create new roles with hand picked actions. The workspace application role is probably ok for Leo but every other resource type probably needs other roles.